crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -2917,7 +2933,10 @@ impl<'db> TypeInferenceBuilder<'db> {
                 _ => Some(Type::Todo),
             },
             // TODO: handle more types
-            _ => Some(Type::Todo),
+            _ => match op {
+                ast::CmpOp::Is | ast::CmpOp::IsNot => Some(KnownClass::Bool.to_instance(self.db)),
```

Comment:
I can remove this again if we prefer to return `Type::Todo` for now, to remind us that we *could* do better in some cases (and return `Literal[True]` or `Literal[False]` instead). But compared to the other operators, it can't be overloaded, so `bool` should always be correct(?).

Reply:
I'm fine with this change; we don't need `Todo` if we are returning a correct and "reasonably precise" (*waves hands*) result -- something that a user isn't likely to consider outright broken. I think this certainly qualifies.